### PR TITLE
notify everybody when a new project is published

### PR DESCRIPTION
**Pull request by g4s8**
#384 - added `pulish_project` bundle test, `notify_all` stakeholder script with `@todo`, call this script from `publish_project`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#541](https://github.com/zerocracy/farm/issues/541) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#541](https://github.com/zerocracy/farm/issues/541) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/421255734). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/resources/com/zerocracy/stk/pm/comm/notify_all.groovy [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ *  This script should send a notification
+ *  to all users, who is not on vacation.
+ *  Let's mention in the notification, that if they don't want to
+ *  receive them anymore, they just have to turn the vacation mode ON.
+ *  Also 'publish_project' bundle test should be fixed.
+ */
+def exec(Project project, XML xml) {
+  new Assume(project, xml).type('Notify all')
+
+  // should be replaced with actual implementation
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/comm/notify_all.groovy]: @g4s8 instead of this comment, just place the ticket inline here.

Comment (g4s8) [reply src/main/resources/com/zerocracy/stk/pm/comm/notify_all.groovy]: @carlosmiranda sure, I moved it into `todo` section. Please look again.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 master with #543 fix was merged, please restart Rultor

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), total is [+1260](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+9700](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#541](https://github.com/zerocracy/farm/issues/541) is now out of scope

**Pull request merged**